django/thunderstore/repository/package_upload.py

```diff
@@ -143,10 +144,14 @@ def save(self, *args, **kwargs):
         self.instance.file_size = self.file_size
         team = self.cleaned_data["team"]
         team.ensure_can_upload_package(self.user)
-        self.instance.package = Package.objects.get_or_create(
-            owner=team,
+        # We just take the namespace with team name for now
+        namespace = Namespace.objects.get_or_create(
```

Comment:
This is created wrong, use the team's name and not the package's name

Reply:
Even better, add a method to the team called `get_namespace` or something which gets the namespace for that team, and handle it there.

Reply:
Also should add a test that ensures a package is created to the correct namespace (so, the team name, instead of the package name in this case).